src/main/groovy/io/seqera/wave/controller/RegistryProxyController.groovy

```diff
@@ -159,7 +164,9 @@ class RegistryProxyController {
         )
     }
 
-    protected void increaseWavePullsCounter(RoutePath route) {
+    protected void increaseWavePullsCounter(RoutePath route, HttpRequest httpRequest) {
+        //fire a pull event
+        eventPublisher.publishEvent(new PullEvent(route.request, addressResolver.resolve(httpRequest)))
```

Comment:
Allow the logic below for the `meterRegistry` can be removed.

Reply:
done